### PR TITLE
[python-sdk] Add ability to download incrementally and skip days.

### PR DESCRIPTION
**Pull request by hurrycane**
Incremental flag - default on - adds the ability to download datasets incrementally.
skip_days - multiple list of strings - adds the ability to skip certain days in the dataset.

### CODE REVIEW

File: moonsense/download.py [hunk 1]
```diff
@@ -155,14 +204,33 @@ def download(
...
+            max_timestamp = self.get_max_timestamp(datadir, with_journey_id)
+            if max_timestamp is not None:
+                max_timestamp = datetime.fromtimestamp(int(max_timestamp))
+                # if we have a max_timestamp, we need to add 1 second to it
+                # so that we don't download the same session again
+                max_timestamp = max_timestamp + timedelta(seconds=1)
+                since = max_timestamp
+        
         if since > until:
+            print(since, until)
```
Comment (andreisavu) [inline moonsense/download.py]: remove?

File: moonsense/download.py [hunk 2]
```diff
@@ -176,30 +244,58 @@ def download(
         session_counter = 0
         filter_by_labels = labels if len(labels) > 0 else None
 
+        max_timestamp_per_day = {}
+
+        print(since, until)
```
Comment (andreisavu) [inline moonsense/download.py]: as an alternative to removing it may be worth using the logging library

**Review by andreisavu — APPROVED** [review]
Looks good to me! I think there are some extra print statements. It may be worth the standard python logging library configured to output on stdout.

File: moonsense/download.py (hunk 2 above)
Comment (hurrycane) [reply moonsense/download.py]: Changed this to proper printing the dates. We'll need to add the logging library at some point for this too but I think we need to do it everywhere so outside of the scope of this change.

**Pull request merged**